githooks/prepare-commit-msg

```diff
@@ -96,6 +96,23 @@ ${cchar}Release note (bug fix): \\
 "
 fi
 
+# Add an issue or epic reference.
+if ! grep -q -E '^(?i:close[sd]?|fix(?:e[sd])?|resolve[sd]?)' "$1"; then
+	sed_script+="/$cchar Please enter the commit message for your changes./i\\
+$cchar Please enter a valid issue or epic reference:\\
```

Comment:
Most of the time our engineers will just know that an issue / epic ref is required and they just need a reminder. So this line doesn't proved a lot of extra value. Let's remove it and instead add text like the following after [this line](https://github.com/cockroachdb/cockroach/blob/421aff1fabcfe16d93ebf2e9169d19fa04c0c997/githooks/prepare-commit-msg#L123):

```
#cchar An issue or epic reference must be in the PR body or each commit message\\
#cchar if the PR or commit is part of an issue or epic. Use `Epic: none` otherwise.\\
#cchar See also: https://wiki.crdb.io/wiki/spaces/CRDB/pages/2009039063/\\
#cchar\\
```